web/thesauruses/java/11/operators.json

```diff
@@ -31,27 +31,48 @@
       "name": "Multiplication and assignment operator"
     },
     "division": {
-      "code": "/",
+      "comment": "If either operand is a float or double, then floating point division is used",
+      "code": "10.0d / 4",
       "name": "Division operator"
     },
     "division_assignment": {
-      "code": "/=",
+      "code": "x /= 2",
       "name": "Division and assignment operator"
     },
+    "integer_division": {
+      "comment": "If both operands are integers or longs, then integral division is used",
+      "code": "10 / 3",
+      "name": "Integer division operator"
+    },
+    "integer_division_assignment": {
+      "code": "x /= 4",
+      "name": "Integer division and assignment operator"
+    },
     "modulus": {
       "code": "%",
       "name": "Modulus (remainder) operator"
     },
     "modulus_assignment": {
       "code": "%=",
       "name": "Modulus and assignment operator"
+    },
+	"unary_plus": {
+      "comment": "This is somewhat redundant as the default sign for numeric values is positive",
+      "code": "x = +7",
+      "name": "Unary plus operator"
+    },
+    "unary_minus": {
+      "code": "y = -12",
+      "name": "Unary minus operator"
     },
     "increment": {
-      "code": "++",
+      "comment": "Supports both pre and post increment",
+      "code": "x++\n++x",
```

Comment:
Your examples are post-increment then pre-increment, but mention it's pre then post. Can you swap one of those so they match?